packages/terra-site/src/site.scss

```diff
@@ -76,6 +76,18 @@
   .site-input-display {
     display: inline-block;
   }
+
+  .modal-opened-content{
```

Comment:
Let's make these all 1 class, `demo-modal-content` and use it across all 3 demos.

Reply:
Updated https://github.com/cerner/terra-core/pull/1048/commits/f06f7a6050a693f3527615fa37892df515eec782